react-components/stories/fileViewers/VideoViewer.stories.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from "react";
+import VideoViewer from "../../src/fileViewers/VideoViewer";
+
+class VideoViewerTest extends Component {
+
+    render() {
+
+        const url = 'https://www.youtube.com/watch?v=ytJb-xDl69U';
```

Comment:
> Video unavailable
> This video has been removed by the user

😞

Reply:
Oh no!  I guess the infamous `LoFi Hip Hop Radio - Beats to Study/Relax To` video changes urls periodically.  I guess I'll have to pick a different one 😄